src/main/java/mate/academy/onlinebookstore/controller/BookController.java

```diff
@@ -27,34 +32,42 @@ public class BookController {
 
     @PostMapping
     @ResponseStatus(HttpStatus.CREATED)
+    @Hidden
```

Comment:
why it hidden?

Reply:
I thought it is documentation for customers